src/main/java/uk/gov/hmcts/reform/sscs/service/ServiceHearingsService.java

```diff
@@ -51,12 +61,24 @@ public ServiceHearingValues getServiceHearingValues(ServiceHearingRequest reques
         String updatedCaseData = objectMapper.writeValueAsString(caseData);
 
         if (!originalCaseData.equals(updatedCaseData)) {
-            log.debug("Updating case data with Service Hearing Values for Case ID {}", caseData.getCcdCaseId());
-            ccdCaseService.updateCaseData(
-                    caseData,
-                    EventType.UPDATE_CASE_ONLY,
-                    "Updating caseDetails IDs",
-                    "IDs updated for caseDetails due to ServiceHearingValues request");
+            if (updatCaseOnlyHearingV2Enabled) {
+                log.debug("Updating case V2 data with Service Hearing Values for Case ID {}", caseData.getCcdCaseId());
+                Long currentCaseId = Long.parseLong(caseData.getCcdCaseId());
+                updateCcdCaseService.updateCaseV2(currentCaseId,
```

Comment:
I think this may be a good candidate for the triggerEventV2 type call, since no caseData update is actually being made?

Reply:
done